src/ts/browserutils.ts

```diff
@@ -25,6 +25,13 @@ export class BrowserUtils {
     return navigator && navigator.userAgent && /Android/.test(navigator.userAgent);
   }
 
+  static get isIOS(): boolean {
+    if (!this.windowExists()) {
+      return false;
+    }
+    return navigator && navigator.userAgent && (/iPad|iPhone|iPod/.test(navigator.userAgent) || (navigator.platform === 'MacIntel' && navigator.maxTouchPoints > 1));
```

Comment:
Please move that `navigator.platform === 'MacIntel' && navigator.maxTouchPoints > 1` to a separate named method. Now looking at this, nobody knows what the actual purpose of this is.